python/openassetio_traitgen/templates/python/traits.py.in

```diff
@@ -24,6 +24,51 @@ class {{ trait.name | to_py_class_name }}Trait(TraitBase):
     """
     kId = "{{ trait.id }}"
 
+    def __init__(self, traitsData):
+        """
+        Construct this trait view, wrapping the given data.
+
+        @param traitsData @fqref{TraitsData}} "TraitsData" The target
+        data that holds/will hold the traits properties.
+        """
+        self._data = traitsData
```

Comment:
[question] I think we originally needed to expose `_data` because `TraitBase` was a base class, so subclasses required access to the data. 

We no longer have that problem, so I wonder if `_data` should become private, i.e. `__data`.

What do you think?  Perhaps this should be a `[plan]`.

Reply:
Yeah sure if you agree it's a good idea, feel free to do it in this PR

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/27/commits/be4a2ab8976156c4d6f344e3665bf4e03b9ffbfc